src/TraceEvent/RegisteredTraceEventParser.cs

```diff
@@ -1076,6 +1078,88 @@ internal struct TRACE_EVENT_INFO
             public EVENT_PROPERTY_INFO EventPropertyInfoArray;  // Actually an array, this is the first element.  
         }
 
+#if NETSTANDARD2_0
```

Comment:
I prefer that the 'going forward' code base have as few #ifdefs as possible (that the #ifdefs are only there for hacks to make older things work).      System.Diagnostics.Eventing.EventDescriptor has not made it into NetStandard 2.0 so we should NEVER rely on it.   Looking around, I see that only use of it seems to be in the definition of TRACE_EVENT_INFO.  Thus the definition should be put there (it could also be simplified to be a simple struct with correct fields of the correct size, which is what we do with all the other interop structs.   This definition should be unconditional.   (it is just another interop struct that happens to be used in RegisteredTraceEventParser.cs